compat/win32/pthread.c

```diff
@@ -57,6 +57,8 @@ pthread_t pthread_self(void)
 	return t;
 }
 
+#ifdef GIT_WIN_XP_SUPPORT
```

Comment:
I fear that my suggestion might have been misguided, and based on a misreading the willingness to contribute of some earlier posts in the same Google group.

I just sent out a final plea for help from Windows XP users to the same group (Cc:ing the Git mailing list), and will see how that unravels.